node/libs/concurrency/src/net/http/mod.rs

```diff
@@ -0,0 +1,46 @@
+//! Http Server configuration structs
+use std::{net::SocketAddr, path::PathBuf};
+
+use anyhow::{Context, Result};
+/// Debug Page credentials (user:password)
```

Comment:
don't derive Debug on a struct with secrets. You can implement Debug manually with the secrets redacted instead.